core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -229,7 +230,16 @@ private TableMetadata createTableMetadata(com.datastax.driver.core.TableMetadata
       ClusteringOrder clusteringOrder = metadata.getClusteringOrder().get(i);
       builder.addClusteringKey(clusteringColumnName, convertOrder(clusteringOrder));
     }
-    metadata.getIndexes().forEach(i -> builder.addSecondaryIndex(i.getTarget()));
+    metadata
+        .getIndexes()
+        .forEach(
+            i -> {
+              // On Cassandra 2.2 only, the indexed column will be in a quoted format so we need to
+              // unquote it. For this purpose, we use the Cassandra SDK `ParseUtils.unDoubleQuote()`
+              // which reverses the effect of `Metadata.quoteIfNecessary()`
+              String indexedColumn = ParseUtils.unDoubleQuote(i.getTarget());
```

Comment:
Suppose the indexed column is quoted because it contains non-alphanumeric characters or reserved keywords. In that case, the value returned by `i.getTarget()` will be : 
- with Cassandra 2.2 : the quoted column name such as `"select_col"`
- with Cassandra 3.X : the unquoted column name such as `select_col`
Since we use the unquoted column name in the ScalarDB `TableMetadata`, we unquote it by using a method of the Cassandra SDK.

Reply:
Indeed, thank you. I am ok with phasing out Cassandra 2.2.
Also, Cassandra 3 EOL is planned for December this year so maybe we should consider supporting Cassandra 4 with ScalarDB 4.0.

Reply:
@brfrn169 Do you see any issues?
If you don't, let's make it unsupported.

Reply:
@feeblefakie @Torch3333 Agreed. let's make Cassandra 2.2 unsupported. Thanks.

Reply:
Understood. 
I will close this PR then.